package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Proof of concept for custom Undertaker registries that pre-define tasks",
   "main": "index.js",
   "scripts": {
-    "test": "lab -cv"
+    "lint": "eslint . && jscs index.js",
+    "pretest": "npm run lint",
+    "test": "echo \"No test specified\""
```

Comment:
This was throwing errors because **lab** isn't a devDep (plus there aren't any tests).